src/backend/cc.d

```diff
@@ -541,13 +552,18 @@ struct block
     int numSucc()                    { return list_nitems(this.Bsucc); }
     block* nthSucc(int n)            { return cast(block*)list_ptr(list_nth(Bsucc, n)); }
 //    void setNthSucc(int n, block *b) { list_ptr(list_nth(Bsucc, n)) = b; }
+
+    static uint sizeCheck();
+    unittest { assert(sizeCheck() == block.sizeof); }
 }
 
 block* list_block(list_t lst) { return cast(block*)list_ptr(lst); }
 
 /** Basic block control flow operators. **/
 
-enum BC {
+alias BC = int;
+enum
```

Comment:
I just saw that the rest of the file uses the same ugly pattern, so I'm fine with leaving it for another time.